### PR TITLE
TS: Bitcoin/Electrum add getNetwork function

### PR DESCRIPTION
**Pull request by nkuba**
The function can be used to compare if the network supported by the
server is the one the user wants. This can be used to find problems with misconfigured electrum API URL on the client start.

Depends on: https://github.com/keep-network/tbtc-v2/pull/504

Refs: https://github.com/keep-network/optimistic-minting/issues/27

### CODE REVIEW

**Review by lukasz-zimnoch — APPROVED** [review]
LGTM! Once #504 is merged, I'm good to go!

**Review by lukasz-zimnoch — APPROVED** [review]

**Review by tomaszslabon — APPROVED** [review]

**Pull request merged**
